docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -325,12 +325,14 @@ A few very important notes about XFF:
      Envoy will not consider it internal. This is a known "bug" due to the simplification of how
      XFF is parsed to determine if a request is internal. In this scenario, do not forward XFF and
      allow Envoy to generate a new one with a single internal origin IP.
-3. Testing IPv6 in a large multi-hop system can be difficult from a change management perspective.
-   For testing IPv6 compatibility of upstream services which parse XFF header values,
-   :ref:`represent_ipv4_remote_address_as_ipv4_mapped_ipv6 <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.represent_ipv4_remote_address_as_ipv4_mapped_ipv6>`
-   can be enabled in the v2 API. Envoy will append an IPv4 address in mapped IPv6 format, e.g.
-   ::FFFF:50.0.0.1. This change will also apply to
-   :ref:`config_http_conn_man_headers_x-envoy-external-address`.
+
+.. The following feature is not implemented yet, preserving the RST for future use.
```

Comment:
Maybe just delete this and the text in the next file? We can always pull it from history.